.github/workflows/tasklist-ci-build-reusable.yml

```diff
@@ -83,10 +83,14 @@ jobs:
           nexusPassword: ${{ steps.secrets.outputs.NEXUS_PSW }}
 
       #########################################################################
-      # Build: maven artifacts and Docker images
-      - name: Build Maven
+      # Build frontend
+      - name: Build frontend
+        uses: ./.github/actions/build-tasklist-fe
```

Comment:
❓ Does it maybe make sense to separate these workflows? And run the frontend only if the frontend changes? 🤔 Or do we E2E have tests were both are covered?

Reply:
we need to build the FE to create the full artifact and build the docker image afterward in the workflow